src/library/navigation/tabs.js

```diff
@@ -0,0 +1,25 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import React from 'react'
+import myDen from '../../containers/Home/layout/myDen';
+import NewDen from '../../containers/Home/layout/NewDen';
+import StackScreen from './stack';
+
+const Tab = createBottomTabNavigator();
+
+export const MyTabs = () => {
+    return (
+        <Tab.Navigator
+            tabBarOptions={{
+                labelStyle: {
+                    fontSize: 15,
+                },
+                activeTintColor: 'black',
+                activeBackgroundColor: '#8CB4D9',
+                inactiveBackgroundColor: '#B9D7F2',
+            }}>
+            <Tab.Screen name='Home' component={StackScreen} />
+            <Tab.Screen name='ND' component={NewDen} />
+            <Tab.Screen name='MD' component={myDen} />
```

Comment:
As a visual code recommendation, it is that the variables have meaningful names and are larger than 3, this in order to quickly identify them in the future. Here I mean the 'name' variables.